docs/general/logging.md

```diff
@@ -12,3 +12,9 @@ You may customize the logger in an initializer, for example:
 ```ruby
 ActiveModelSerializers.logger = Logger.new(STDOUT)
 ```
+
+You can also disable the logger, just put this in `config/application.rb`:
+
+```ruby
+ActiveSupport::Notifications.unsubscribe(ActiveModelSerializers::Logging::RENDER_EVENT)
```

Comment:
It's a better practice to put this in an initializer in a Rails app.

`config/initializers/active_model_serializers.rb` is the usual location.

```ruby
require 'active_model_serializers'
ActiveSupport::Notifications.unsubscribe(ActiveModelSerializers::Logging::RENDER_EVENT)
```

and perhaps reference the relevant railtie https://github.com/rails-api/active_model_serializers/blob/0-10-stable/lib/active_model_serializers/railtie.rb#L23-L27 ?